### ISSUE TITLE
Please add Andrew G. Dick as contributor

### ISSUE THREAD

**Issue by DonaldMcIntyre** [ethereumclassic/ethereumclassic.github.io#1039]
This issue is to request to add Andrew G. Dick, marketing manager at the ETC Cooperative, so he may review and approve new article PRs to provide some admin redundancy to make sure we can publish our posts on time.

His GitHub profile is:

https://github.com/andrewgdick

**Comment by w1g0** [comment]
If we are going to allow more ETC coop reviewers, shouldn't we increase the number of reviews as well?
It's true that from the independent reviewers there's not always the swiftest response and that needs to be improved.
ETC Cooperative reviews the articles internally so ideally we would like to peer review before publishing on the website.

**Comment by DonaldMcIntyre** [comment]
That is fair and reasonable. I also think that if ETC Coop writes and reviews and approves its own posts then it loses the checks and balances. 

This was just an idea for the Christmas and New Year's season as several independent admins may be traveling and visiting family.

So, I will close this PR and keep insisting over social media and here to admins to review the posts so we don't fall behind with the weekly schedule (1 content unit each Tuesday, Wednesday, and Thursday).

Thank you!

**Comment by TheCrowbill** [comment]
@DonaldMcIntyre please reopen this issue. It will give the other maintainers a chance to catch up or comment.

Also, not all of us can add new members to the organization. I think that is set for only the organization's owner and admins.

**Comment by DonaldMcIntyre** [comment]
> @DonaldMcIntyre please reopen this issue. It will give the other maintainers a chance to catch up or comment.
> 
> Also, not all of us can add new members to the organization. I think that is set for only the organization's owner and admins.

Reopened.

**Comment by TheCrowbill** [comment]
Sent from Yahoo Mail on Android 
 
  On Tue, Dec 20, 2022 at 1:55 PM, Donald ***@***.***> wrote:   



@DonaldMcIntyre please reopen this issue. It will give the other maintainers a chance to catch up or comment.

Also, not all of us can add new members to the organization. I think that is set for only the organization's owner and admins.


Reopened.

👍

**Comment by bobsummerwill** [comment]
What are the current required reviewer counts for website and Twitter?

**Comment by bobsummerwill** [comment]
More non-Coop reviewers would be great, if we can find more volunteers.   I would be reticent to raise the required counts **too** high, though, because it introduces real friction.

**Comment by bobsummerwill** [comment]
In terms of fears about “capture” it is worth remembering that with this review process there is no potential here for taking over any administrative functions or doing anything very bad.  Reviewers are not super-users.  The worst they can do it publish “bad stuff”, but there is a clear papertrail to demonstrate that abuse and it is easily rectified.

**Comment by bobsummerwill** [comment]
We absolutely do not want a situation where the Coop is just “pushing to prod” blindly with quality gate from the rest of the community.

Just to clarify - is there a specific bounded group of accounts which are able to progress a review?  Or can any GitHub account do so?

**Comment by bobsummerwill** [comment]
If we cannot get these articles reviewed in time with acceptable “neutrality” then maybe we just have some days without content?   We have had weeks or months without new content for the Coop in the past, so it would not be a disaster.

Step away from the keyboards, ETC, and spend some time with your families! 😀

CC @bubblegumtate

**Comment by TheCrowbill** [comment]
> What are the current required reviewer counts for website and Twitter?

For the website, two.
For eth_classic Twitter, two.
For etc_network Twitter, one.

The only requirements on reviewers are membership in the organization and write access. A reviewer may be specifically requested by a PR's author, but if no requests, any of us with write access suffice.

**Comment by bobsummerwill** [comment]
Right, thanks.

So there is an approval process here where the admins of the ethereumclassic org would need to invite specific accounts to join the org?  But we have a potentially unbounded amount of those, and they can do little damage.

**Comment by TheCrowbill** [comment]
Yes there is a process but it is low impact. Organization guidelines stipulate that in order to be given the necessary access, one must:

- have materially contributed to the ETC protocol or community. (How is not mandated.)
- have 2FA setup on your GitHub account.
- make oneself known as an anti-Sybill measure. (Again, the method for this is not specific.)

As to the potential for damage, I don't think anyone is suggesting the Cooperative is planning to harm the protocol or the community. That said, this community remembers the past as we watched trusted stewards become bad actors. I don't care to make our missteps over again.

There is no downside to having more editorial eyes on a project prior to publishing. So long as the minimum number of required reviews remains reasonable, raising them by one or two would be more beneficial than burdensome.

**Comment by bobsummerwill** [comment]
"That said, this community remembers the past as we watched trusted stewards become bad actors. I don't care to make our missteps over again."

100%.

We need to be doing everything we can be as systemically resilient to internal and external social attacks as we can.

To my mind, our largest remaining weak spots all lie where we use centralized platforms which have all-powerful "admin accounts" where multiple individuals all have super-powers, including over each other.  Examples being GitHub (ethereumproject org) and Twitter (eth_classic).  I see no good solutions there unless the platforms themselves change to provide multi-sig style admin support.  Discord has the same weakness.

Short of that, we only have redundancy on those platforms as a (weak) mitigation.

**Comment by w1g0** [comment]
> In terms of fears about “capture” it is worth remembering that this review process has no potential for “taking over” any administrative functions or doing anything very bad. Reviewers are not super-users. The worst they can do it publish “bad stuff”, but there is a clear paper-trail to demonstrate that abuse and it is easily rectified after the fact.

This is not about fear of a capture, it's mainly about how peer review can improve the content of the already high value content that @DonaldMcIntyre is producing. Also very often mistakes or unclarities seep in because of the elevated knowledge of the specialist reviewers. Sometimes theories are taken for fact and should be modified before publication. Once published the content gets spread to different platforms and the stories start to live their own lives. Undoing is a waste of energy and I'm a proponent of peer review.

Look this is only my reflection, everyone can tune it, I'm just a voice in this ecosystem.

**Comment by bobsummerwill** [comment]
I would agree on getting the content right prior to publication, @w1g0.   Review is very important.

I think that @DonaldMcIntyre's "let get this out and then we'll fix little things later" (which I saw somewhere and cannot find now - maybe in review for one of the pending PRs) was actually more related to a bug/issue in the Netlify workflow where simple changes get stuck in the preview build and we are not sure how they get unstuck.

Looks like this:
https://app.netlify.com/sites/jocular-bienenstitch-b37dca/deploys/63a23ec7efdcc60009fd884b

**Comment by bobsummerwill** [comment]
Thanks, @IstoraMandiri.

With regard to adding Andrew, I am deliberately trying to remove myself from the loop on as much stuff as possible anyway, so it would functionally be more of a matter of Andrew replacing me.   The whole point of having a comms team is for me to not be needed.

It is certainly preferable to have non-Coop people reviewing wherever possible.

I would agree on that negative to an additional reviewer.  Things are slow as it is.

**Comment by w1g0** [comment]
> I would agree on that negative to an additional reviewer. Things are slow as it is.

Convenience is the Achilles' heel of security.
